hw-model/src/model_fpga_realtime.rs

```diff
@@ -446,9 +449,11 @@ impl HwModel for ModelFpgaRealtime {
     }
 
     fn output(&mut self) -> &mut crate::Output {
-        self.output
-            .sink()
-            .set_now(self.start_time.elapsed().as_millis().try_into().unwrap());
+        self.output.sink().set_now(
```

Comment:
In a future PR, we should probably consider exposing an actual cycle count register from the FPGA testbench RTL.

Reply:
I agree. I don't love the assumption about the FPGA speed I had to make here.
Should be easy to do.